### ISSUE TITLE
Electron fails to start / load page when running under Windows 7, starting with version 5.0.12 and persisting through 7.1.7, and starting again with 8.2.2 and persisting with 8.2.3

### ISSUE THREAD

**Issue by ccurtis0** [electron/electron#21544]
### Preflight Checklist

* [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [X] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * 5.0.12 << 7.1.9; also 8.2.2
* **Operating System:**
  * Windows 7 (Version 6.1.7601)
* **Last Known Working Electron version:**
  * 5.0.11; also 8.2.1
* **Node version:**
  * v12.13.1
* **NPM version:**
  * 6.13.4; also 6.14.4

### Expected Behavior
I expect the Electron application to launch and load the given page/directory.

### Actual Behavior
The application loads and shows an empty/blank/white window, but then exits before displaying any content or completing any outstanding operations (such as a timer set to prevent exiting).

### To Reproduce
From a Windows 7 machine, follow the electron quick-start:
* ```git clone https://github.com/electron/electron-quick-start```
* ```npm install```
* ```npm start```

### Additional Information
The gory details are [on Stack Overflow](https://stackoverflow.com/questions/59254682/electron-application-exits-immediately-after-starting), so I will not repeat them here.

To summarize: I have an application that was working at Electron version '^5'. I upgraded to Electron version '^7' and it failed to start. I downgraded to '^5', which installed @5.0.12 and continued to fail. I found that version @5.0.11 works as it did before. The same problem is exhibited with the electron-quick-start application, which can be used for testing. For more details, the StackOverflow link shows my additional debug efforts.

Please let me know what additional information you need. I've removed all the global npm modules mentioned in the link; everything is being run from the local directory. 2019/12/20: I see a new electron version was released (7.1.7) so I did a fresh checkout of the quickstart demo and verified that it continues to fail.

**Comment by chill-patel** [comment]
We are also facing same issue in windows 7 only in windows 10 its works fine.

**Comment by mehulsutariya** [comment]
Is there any update on this issue? We are facing the same problem and we are trying to migrate to Electron 7, however this issue is blocking us from doing so. It would be great if someone can provide an update.

**Comment by ccurtis0** [comment]
I upgraded to Electron 7.1.9 to test if this problem persists. The post-install seemed to run an unusually long time, but my application started and seemed to work fine. I downgraded to 7.1.8 and that also ran fine. I downgraded again to 7.1.7, which was failing, and now seems to work as well.

If this is working for other folks, we may be able to close this. Nothing in the release notes seems applicable, except maybe the crash reporter change in 7.1.8. If anyone is interested, see if 7.1.7 fails and if upgrading to 7.1.8 fixes it.

![image](https://user-images.githubusercontent.com/1854842/72907736-6ad82300-3d02-11ea-8e25-9e3fc90a929c.png)

**Comment by ccurtis0** [comment]
> Is there any update on this issue? We are facing the same problem and we are trying to migrate to Electron 7, however this issue is blocking us from doing so. It would be great if someone can provide an update.

Are you still having this problem? Did you try upgrading to 7.1.9+?

**Comment by ccurtis0** [comment]
Today, feeling confident, I upgraded from 8.2.1 to 8.2.2.

However, my application was running in the debugger (VSCode) and the upgrade failed. I attempted the upgrade ('ncu -u'; 'npm i') a couple times before I realized it was failing. In all cases, Electron was the only module being upgraded. I suspect this is relevant to the problem.

I tried to downgrade to 8.2.1 and other 7.x versions but the problem followed. After many struggles, I realized that the post-install was always run from electron@8.2.2. I believe this was because of package-lock.json. I deleted package-lock.json and reinstalled electron@8.2.1 from npm, but the post-install still used @8.2.2 for reasons I still do not understand. So after deleting node_modules I edited package-lock.json to specify 8.2.1 (and deleted the "Integrity" field) and finally have 8.2.1 installed and working. Now, after upgrading to 8.2.2, my application still behaves as described above, but I can downgrade to 8.2.1 and continue development.

**Comment by ccurtis0** [comment]
Short version: I realized my problem with package-lock.json was that I was using ^8.2.1 instead of =8.2.1. I ensured the debugger wasn't running and tried upgrading to 8.2.3. No errors were emitted by npm, but the troublesome behavior returned: a white window would open for a fraction of a second and then the application would exit. However, going to electron =8.2.1 no longer worked. I deleted my application and electron and npm.cache folders from %USERDATA%\Local and Roaming, and installed every from 8.2.3 to the last couple versions in the 6.x series, with none of them working.

Running with --v=1 and --enable-logging provided no help, but I was able to get it running with the "--no-sandbox" argument. This is how I am currently forced to use electron 8.2.1 to run my application.

**Comment by tobemaster56** [comment]
i meet with this problem too

**Comment by electron-triage** [comment]
The Electron version reported on this issue is no longer supported. See our [supported versions documentation](https://www.electronjs.org/docs/tutorial/support#supported-versions).

**If you're still experiencing this issue on a supported version, please open a new issue with an updated repro - a [Fiddle](https://www.electronjs.org/fiddle) is very appreciated.**

Electron has a large issues backlog. To help our team prioritize, we're closing older issues and asking for new issues with updated repro steps if it affects a supported version. This helps sort what issues are still relevant and helps us fix them more quickly.

Thanks for your patience and understanding!
